daily-challenges/2024-10/31/lcaloi.29.spec.ts

```
@@ -0,0 +1,72 @@
+type Candy = {
+    type: string;
+    quantity: number;
+};
+
+type TrickOrTreater = {
+    name: string;
+    candies: Candy[];
+};
+
+function findTopTrickOrTreater(trickOrTreaters: TrickOrTreater[]): string {
+    let name: string = '';
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)